components/layout/PatternsLayout.tsx

```diff
@@ -0,0 +1,99 @@
+  </header>
+);
+
+const PatternNav = (props: { patternClasses: PatternClass[] | undefined, onClick: Dispatch<SetStateAction<PatternClass | undefined>>, selectedPatternClass: PatternClass | undefined }) => (
+    <nav className="columns-8 h-10 gap-0">
+      {props.patternClasses && props.patternClasses.map(patternClass =>
+        <button
+          key={patternClass.name}
+          className={`block w-full h-full text-left pl-2 text-sm hover:underline ${patternClass.name === props.selectedPatternClass?.name ? "bg-white" : patternClassLookup[patternClass.name]}`}
+          onClick={() => props.onClick(patternClass)}
+        >
+          {patternClass.name}
+        </button>
+      )}
+    </nav>
+);
+
+const PatternList = (props: { patternClass: PatternClass | undefined }) => {
+  const { patternClass } = props;
+  const { data: patternInfo, error: patternInfoError } = trpc.patternInfo.useQuery({ patternClassName: patternClass?.name || null })
+
+  return (
+    <section className="p-8">
+      <p className="mb-4">{patternClass?.description}</p>
+      <h3 className="text-lg mb-4">Rights</h3>
+      {patternInfo?.rights.map(pattern => (
+        <PatternItem key={pattern.name} pattern={pattern} patternClassName={patternClass?.name} highestCount={patternInfo?.rights[0].entryCount} />
+      ))}
+      <h3 className="text-lg mb-4">Obligations</h3>
+      {patternInfo?.obligations.map(pattern => (
+        <PatternItem key={pattern.name} pattern={pattern} patternClassName={patternClass?.name} reverse highestCount={patternInfo?.obligations[0].entryCount} />
+      ))}
+    </section>
+  )
+}
+
+const PatternItem = (props: { pattern: Pattern, patternClassName: string | undefined, reverse?: boolean, highestCount: number | undefined }) => {
+  const { pattern, patternClassName, reverse, highestCount } = props
+
+  return (
+    <div className={`flex mb-4 ${reverse ? "flex-row-reverse" : ""}`}>
+      <div className={clsx(`w-1/2 ${patternClassLookup[patternClassName!]} bg-opacity-40 flex items-center justify-center`)}>
+        <Hotel size={32} className="w-1/4 flex-center" />
+        <div className="p-4 w-3/4">
+          <p className="text-lg">{pattern.name}</p>
+          <p className="text-xs">{pattern.description}</p>
+        </div>
+      </div>
+      <div className="w-1/2 grid" style={{ gridTemplateColumns: `repeat(${highestCount}, minmax(0, 1fr))`, direction: reverse ? "rtl" : "ltr" }}>
+        <div className={clsx(`${patternClassLookup[patternClassName!]} bg-opacity-20 flex justify-center flex-col py-2 ${reverse ? "pr-4" : "pl-4"}`)} style={{ gridColumn: `span ${pattern.entryCount}` }}>
```

Comment:
also: clever uses of `reverse` here! i was fiddling with something similar for the charts for awhile but never quite landed on it - definitely going to come back and reference this when i clean up chart labels :slightly_smiling_face: